pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -392,10 +490,19 @@ def newpkgcb(pkgId, name, arch):
                             dc.extra_data = future_relations
                             await self.put(dc)
 
+            # now save modules as should have all relations set up
```

Comment:
s/save modules as should have all relations set up/send modules down the pipeline since all relations have been set up/